CONTRIBUTING.md

```diff
@@ -10,6 +10,68 @@ Some of the core team will be working directly on GitHub. These changes will be
 
 We will do our best to keep `master` in good shape, with tests passing at all times. But in order to move fast, we will make API changes that your application might not be compatible with. We will do our best to communicate these changes and version appropriately so you can lock into a specific version if need be.
 
+### Commit Message Prefix
+
+The header has a prefix:
+
+````
+<prefix> <subject>
+<BLANK LINE>
+<body>
+````
+
+ * [ActivityIndicator]
+ * [AdsManager]
+ * [Assets]
+ * [Bridge]
+ * [CLI]
+ * [CameraRoll]
+ * [Children]
+ * [Cosmetic]
+ * [Docs]
+ * [ErrorMessage]
+ * [Errors]
+ * [FBRhinos]
+ * [Flow]
+ * [GeoLocation]
+ * [Haste]
+ * [Image]
+ * [Images]
```

Comment:
I thought Images was not relevant to `Libraries/Image`. Because d0d774d had modified `RCTConvert.m`. It seems my mistake. I have to delete `Images` too.